pkg/routers/api.go

```diff
@@ -130,4 +132,12 @@ func WithAPIRouter(opt *APIOptions, r chi.Router) {
 	})
 
 	r.With(createAccesstokenTimer).Post("/oauth/token", opt.oAuthHandler)
+
+	statikFS, err := fs.New()
+	if err != nil {
+		panic(err)
+	}
+
+	staticServer := http.FileServer(statikFS)
+	r.Handle("/openapi.yaml", staticServer)
```

Comment:
Is this a conventional place to look for "what is the API spec of this service"? If not, is there such a place (something akin to https://en.wikipedia.org/wiki/List_of_/.well-known/_services_offered_by_webservers but for OpenAPI)? If not, would we be willing to serve it on "/" (index)? Seems a little circular that I have to "just know" this one route in order to discover the rest of the routes I can use.

Reply:
From https://swagger.io/specification/
> It is RECOMMENDED that the root OpenAPI document be named: openapi.json or openapi.yaml.

I also found other examples in their [documentation](https://swagger.io/docs/specification/api-host-and-base-path/#relative-urls) like:
> For example, if the definition hosted at http://localhost:3001/openapi.yaml...

Reply:
That being said I think it's a great idea to have an index "/" handler that could say "hello" and link to the spec. I'll add that as it was also mentioned by @asaschachar as a nice to have to improve the overall "landing page" experience.